src/main/kotlin/lotto/LottoMachine.kt

```diff
@@ -0,0 +1,17 @@
+package lotto
+
+private const val TICKET_COST = 1000
+
+class LottoMachine(val money: Money) {
+    val tickets: List<Lotto> = issueTickets()
+
+    fun issueTickets(): List<Lotto> {
+        val ticketCount: Int = money.amount / TICKET_COST
+
+        return (1..ticketCount).map {
+            Lotto(generateRandomNumber())
+        }
+    }
```

Comment:
how about this? 😉 
base on constructor(with `init`)

```suggestion
class LottoMachine(money: Money) {
    val tickets: List<Lotto>

    init {
        val ticketCount: Int = money.amount / TICKET_COST
        tickets = List(ticketCount) { Lotto(generateRandomNumber()) }
    }
```